src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -9459,19 +9458,32 @@ void VRFTerminalUnitEquipment::ControlVRFToLoad(EnergyPlusData &state,
 
     if ((VRFHeatingMode || HRHeatingMode) || (VRFCoolingMode || HRCoolingMode)) {
 
-        Par(1) = VRFTUNum;
-        Par(2) = 0.0;
-        if (state.dataHVACVarRefFlow->VRFTU(VRFTUNum).isSetPointControlled) Par(2) = 1.0;
-        Par(4) = 0.0; // fan OpMode
-        if (FirstHVACIteration) {
-            Par(3) = 1.0;
-        } else {
-            Par(3) = 0.0;
-        }
-        //    Par(4) = OpMode
-        Par(5) = QZnReq;
-        Par(6) = OnOffAirFlowRatio;
-        General::SolveRoot(state, ErrorTol, MaxIte, SolFla, PartLoadRatio, PLRResidual, 0.0, 1.0, Par);
+        auto f = [&state, VRFTUNum, FirstHVACIteration, QZnReq, OnOffAirFlowRatio](Real64 const PartLoadRatio) {
```

Comment:
Sure, I think in a growing number of cases, we can go further and just capture `this` rather than capturing the index and getting it back off of state.